pytest_fixtures/component/repository.py

```diff
@@ -91,6 +91,30 @@ def repo_setup():
     yield details
 
 
+@pytest.fixture(scope='module')
+def setup_content(module_org):
+    """This fixture is used to setup an activation key with a custom product attached. Used for
+    registering a host
+    """
+    org = module_org
+    custom_repo = entities.Repository(
+        product=entities.Product(organization=org).create(),
+    ).create()
```

Comment:
can we have this as function in api_factory as it uses nailgun entities instead of fixture? we could've access this with sat object and this can be used with any scoped satellite objects including destructive, wdyt?

Reply:
This fixture is pretty unique and is specific for the custom repository tests that I wrote. I do not think its necessary to move this into api_factory

Reply:
Agree with @ColeHiggins2. But I believe that @Gauravtalreja1 meant that
```python
custom_repo = entities.Repository(
    product=entities.Product(organization=org).create(),
).create()
```
could be a helper function. Correct, Gaurav?
If that is the case, @ColeHiggins2 do you see this helper being used somewhere else than in this test?